{{ cookiecutter.__repo_name }}/README.md

```diff
@@ -11,6 +13,23 @@ pip install -e .
 deepaas-run --listen-ip 0.0.0.0
 ```
 
+## Contributing
+
+This module tries to enforce best practices by using [Black](https://github.com/psf/black)
+to format the code.
+
+For an optimal development experience, we recommend installing the VScode extensions
+[Black](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter)
+and [Format on Auto Save](https://marketplace.visualstudio.com/items?itemName=BdSoftware.format-on-auto-save).
```

Comment:
Check if **BdSoftware** is needed? 
I run it in my local and it was all working without it.
If we do not add **BdSoftware** then I would remove this line.

Reply:
In my case, formatting did not happen without BSsoftware extension.
I'm using VScode 1.92.1 on Ubuntu 20.04. As far as I can tell, people [still complain it not working](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#minimal-configuration).

Reply:
I have 1.92.2 and U 22.04, however, It should be the same.
I was not trying exactly the same but rather the config here (updated):

https://github.com/ai4os-hub/demo-advanced/blob/main/.vscode/settings.json

Reply:
I tested with your configuration and it still does not work in my local. So to be on the save side, I would just keep the BSsoftware extension.